ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -424,15 +424,41 @@ tests:
     - chain: cucushift-installer-check-cluster-health
   timeout: 4h0m0s
 - always_run: false
-  as: e2e-aws-ovn-user-provisioned-dns
+  as: aws-custom-dns-private
```

Comment:
If I remember right, @gpei suggested to run a public custom-dns cluster using QE's profile, right ?

Reply:
yes, I just submit https://github.com/openshift/release/pull/66973 to allow the `aws-qe` profile to be used in the installer/ repo. 
The workflow of installing a public cluster is more straightforward and the steps are simpler.

Reply:
and it should be more convenient to directly define a new workflow to add custom-dns jobs in pre-submit and release, so don't have to repeat these steps in the two job definitions

Reply:
Updated